### ISSUE TITLE
insertValue and insertTemplate for invisible fields

### ISSUE THREAD

**Issue by hk298** [tabalinas/jsgrid-php#12]
Sorry, to bug you again...

I'm trying to build a table with two columns where the user can edit one column and where the value of the other one is determined by something else in the code. Here's a code snippet that basically does what I want:
[http://jsfiddle.net/bgpfnkfL/3/]
However, I'd like to set _visible: false_ for Column 2. When I do that, it's inserting only 0 into the database, not the value set in the code. Am I still missing something, or is this just how it is?

**Comment by tabalinas** [comment]
Why do you need this field if you hide it? Add necessary field in the `controller.insertItem` before sending an item to the server.

**Comment by hk298** [comment]
Oh, that was easy. Thanks.

I need the hidden field because I have two tables showing data that is in a 1:n relationship. The user selects an item in the first table which then filters the second table on the hidden column.

I was just stuck with the templates. When I used only the `insertValue` function, the grid would allow the user to inter a value but it would use the one from the function. Slightly irritating. When I used only the `insertTemplate`, the table header looked good, but jsGrid threw an error. The jsFiddle does that too. Only when I used both functions I would get a behavior that made sense to me.
